scss/style.scss

```diff
@@ -0,0 +1,721 @@
+      border-right: 0;
+      transform: rotate(-45deg);
+      transition: transform 0.8s ease-in-out;
+      cursor: pointer;
+      &:hover {
+        border-bottom: 1.5px solid $hoverColorLink;
+        border-left: 1.5px solid $hoverColorLink;
+        transition: 800ms border-left;
+        transition: 800ms border-bottom;
+      }
+    }
+    .rotate {
+      transform: rotate(135deg);
+      transition: transform 0.8s ease-in-out;
+    }
+
+    .question {
+      width: 100%;
+      color: #191919;
+      font-family: Aclonica;
+    }
+  }
+}
+
+.footer {
+  background-color: #f2f2f2;
+  width: 100%;
+  font-family: Aclonica;
+  .footer-wrapper {
+    .about-company {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      .link-company {
+        list-style: none;
+        display: flex;
+        flex-direction: column;
+        padding: 0;
+        margin: 24px 0 0 23px;
+        .link-company-item{
+          .link-company-link{
+            font-size: 12px;
+            line-height: 24px;
+            color: $hoverColorLink;
+            margin-top: 12px;
+            text-decoration: none;
+          }
+          #Call{
+            span{
+              font-size: 16px;
+              line-height: 24px;
+              color: $hoverColorLink;
+              margin-bottom: 20px;
+            }
+          }
+        }
+      }
+    }
+    .address-container {
+      padding: 0;
+      margin-left: 23px;
+      .address {
+        display: flex;
+        flex-direction: column;
+        padding: 8px 0 8px 0;
+        border-top: 2px solid $colorTextlinks;
+        .address-title{
+          font-size: 16px;
+          line-height: 36px;
+          color: $hoverColorLink;
+          margin: 0;
+        }
+        .address-text{
+          font-size: 16px;
+          line-height: 24px;
+          color: $hoverColorLink;
+          font-style: normal;
+        }
+        .phone {
+          font-size: 16px;
+          line-height: 24px;
+          color: $hoverColorLink;
+          text-decoration: none;
+        }
+
+        .view-map {
+          text-decoration: none;
+          font-size: 12px;
+          line-height: 20px;
+          color: $colorTextlinks;
+          &:hover {
+            color: $hoverColorLink;
+            transition: 800ms color;
+          }
+        }
+      }
+    }
+  }
+  .social {
+    border-top: 1.5px solid #b2b2b2;
+    margin-top: 12px;
+    padding-top: 20px;
+    text-align: center;
+
+    .conteiner-social-icons {
+      display: flex;
+      justify-content: space-evenly;
+      margin-bottom: 20px;
+      padding-left: 0;
+      .twitter {
+        cursor: pointer;
+        display: inline-block;
+        width: 30px;
+        height: 24px;
+        background-image: url(../images/twitter.png);
+        background-position: 0 0;
+        &:hover {
+          background-position: 30px 0;
+          transition: 300ms background-position;
+        }
+      }
+      .facebook {
```

Comment:
you dublicate a lot of styles for each icon? create commont class for each item. read about DRY principals

Reply:
done